src/searchbar.cpp

```diff
@@ -119,7 +119,7 @@ bool SearchBarLineEdit::eventFilter(QObject *, QEvent *event)
 {
     if (auto e = static_cast<QKeyEvent *>(event))
     {
-        if (e->key() == Qt::Key_PageDown && m_scrolledEndBefore)
+        if (e->key() == Qt::Key_PageDown && m_scrolledEndBefore && !m_noMoreSuggestion)
```

Comment:
Double negation can be confusing. Please replace the `m_noMoreSuggestion` data member with the inverse variable `m_moreSuggestionsAreAvailable`

Reply:
Done